### PR TITLE
Polish changes

### PR DESCRIPTION
**Pull request by vishalbollu**
---

checklist:

- [ ] run `make test` and `make lint`
- [ ] test manually (i.e. build/push all images, restart operator, and re-deploy APIs)
- [ ] update examples
- [ ] update docs and add any new files to `summary.md` (view in [gitbook](https://docs.cortex.dev/v/master) after merging)
- [ ] cherry-pick into release branches if applicable
- [ ] alert the dev team if the dev environment changed

### CODE REVIEW

File: pkg/workloads/cortex/client/cortex/client.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def __init__(self, env: str):
         self.env = env
 
     # CORTEX_VERSION_MINOR x5
-    def deploy(
+    def create_api(
```
Comment (deliahu) [inline pkg/workloads/cortex/client/cortex/client.py:47]: do docs need to be updated for this?

File: pkg/workloads/cortex/client/cortex/binary/__init__.py [hunk 2]
```diff
@@ -67,26 +67,28 @@ def run_cli(
...
-            if result_found:
-                output = output[:-1]
-                if c == "\n":
-                    result_found = False
+            if processing_result:
+                result += c
+                if (
+                    result[: len(MIXED_CORTEX_MARKER)] == MIXED_CORTEX_MARKER
+                    and result[-len(MIXED_CORTEX_MARKER) :] == MIXED_CORTEX_MARKER
+                    and len(result) > len(MIXED_CORTEX_MARKER)
```
Comment (deliahu) [inline pkg/workloads/cortex/client/cortex/binary/__init__.py:83]: Does this address the extra new lines that are printed? We can discuss it live too

**Review by deliahu — APPROVED** [review]
A couple controversial changes in this one :) LGTM :+1:

**Review by deliahu — APPROVED** [review]

**Comment by vishalbollu** [comment]
Update cli/cmd/patch.go to do fmt.Print() for the json output.

**Pull request merged**